docs/source/interface.rst

```diff
@@ -5,18 +5,18 @@
 Interpreter Interface
 =========================
 
-The Scilla interpreter executable provides a calling interface that enables
-users to invoke transitions with specified inputs and obtain outputs. Execution
-of a contract with supplied inputs will result in a set of outputs and a change
-in the smart contract mutable state. 
+The Scilla interpreter provides a calling interface that enables users
+to invoke transitions with specified inputs and obtain
+outputs. Execution of a transition with supplied inputs will result in a
+set of outputs, and a change in the smart contract mutable state.
 
 Calling Interface
 ###################
 
-A transition defined in a smart contract can be called either by the issuance
-of a transaction or by message calls from another smart contract. The same
-calling interface will be used to call the contract via external transactions
-and inter-contract message calls.
+A transition defined in a contract can be called either by the
+issuance of a transaction, or by message calls from another
```

Comment:
Is "issuance" the most appropriate word here? Maybe just "can be called either because of a transaction, or by ..." is sufficient.